### DISCUSSION TITLE
guide/1%20%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80/

### DISCUSSION THREAD

**Discussion by giscus** [kangkaisen/olap-performance#1] [Announcements]
# guide/1%20%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80/

介绍 我们支持 bar 功能，... 详情 baz (baz.md); ...;

http://localhost:8080/guide/1%20%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9F%BA%E7%A1%80/

**Comment by kangkaisen** [comment]
test
